Lib/test/test_selectors.py

```diff
@@ -175,6 +175,31 @@ def test_modify(self):
         self.assertFalse(s.register.called)
         self.assertFalse(s.unregister.called)
 
+
+    def test_modify_unregister(self):
+        # Make sure the fd is unregister()ed in case of error on
+        # modify(): http://bugs.python.org/issue30014
+        if self.SELECTOR.__name__ == 'EpollSelector':
+            patch = unittest.mock.patch('selectors.select.epoll')
+        elif self.SELECTOR.__name__ == 'PollSelector':
+            patch = unittest.mock.patch('selectors.select.poll')
+        elif self.SELECTOR.__name__ == 'DevpollSelector':
+            patch = unittest.mock.patch('selectors.select.devpoll')
+        else:
+            return unittest.skip("")
```

Comment:
`unittest.skip()` is a decorator. Using `raise self.skipTest()` or `raise unittest.SkipTest()` would be more appropriate for this use case.